File: C++/0069. a==b?.cpp
// we have defined the necessary header files here for this problem.
// If additional header files are needed in your program, please import here.

int main()
{  
  // please define the C++ input here. For example: int a,b; cin>>a>>b;;  
  // please finish the function body here.  
  // please define the C++ output here. For example:cout<<____<<endl; 
    string s1,s2;
    while(cin>>s1>>s2){
        for(int i=s1.size()-1;i>=0;--i){
            if(s1[i]!='0' && s1[i]!='.'){
                s1 = s1.substr(0,i+1);
                break;
            }
        }
        for(int i=s2.size()-1;i>=0;--i){
            if(s2[i]!='0' && s2[i]!='.'){
                s2 = s2.substr(0,i+1);
                break;
            }
        }
        cout<<(s1==s2?"YES":"NO")<<endl;
    }
    return 0;
}

File: Python/0069. a==b?.py
# If you need to import additional packages or classes, please import here.

def func():
    # please define the python3 input here. 
    # For example: a,b = map(int, input().strip().split())
    # please finish the function body here.
    # please define the python3 output here. For example: print().
    while True:
        try:
            a,b = map(str,input().strip().split())
            
            for i in range(len(a)-1,-1,-1):
                if a[i] != '0' and a[i] != '.':
                    a = a[:i+1]
                    break
            for j in range(len(b)-1,-1,-1):
                if b[j] != '0' and b[j] != '.':
                    b = b[:j+1]
                    break
            
            print("YES" if a==b else "NO")        
        except:
            break
    
if __name__ == "__main__":
    func()
